src/App.test.js

```diff
@@ -1,8 +0,0 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
-
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
-});
```

Comment:
- [ ] Kindly remove this file and the other ones marked in the image below as they are boilerplate files that won't be used.

  ![image](https://user-images.githubusercontent.com/36542986/234492600-bd46f9a2-1cac-4b82-9b1c-15a9ad85618f.png)

Reply:
Hello @TedLivist 

Thank you for your feedback. I shall make the required changes and ask for a review again.